testData/solutions/win32.json5

```diff
@@ -1,4 +1,6243 @@
+                            "description": "Selects the size of the cursor locator.",
+                            "enum": [
+                                1,
+                                2,
+                                4
+                            ],
+                            "enumLabels": [
+                                "Thin",
+                                "Medium",
+                                "Thick"
+                            ],
+                            "default": 2
+                        }
+                    },
+                    "CursorScheme.CursorEnhColor": {
+                        "schema": {
+                            "title": "Cursor locator color",
+                            "description": "Selects the color to use for the cursor locator.",
+                            "enum": [
+                                255,
+                                65535,
+                                65280,
+                                16776960,
+                                16711680,
+                                16711935
+                            ],
+                            "enumLabels": [
+                                "Red",
+                                "Yellow",
+                                "Green",
+                                "Cyan",
+                                "Blue",
+                                "Magenta"
+                            ],
+                            "default": 16711680
+                        }
+                    },
+                    "CursorScheme.CursorEnhTransparency": {
+                        "schema": {
+                            "title": "Cursor locator transparency",
+                            "description": "Specifies the cursor locator transparency to use. From 'Clear' to 'Solid'.",
+                            "type": "integer",
+                            "minimum": 31,
+                            "maximum": 255,
+                            "default": 143
+                        }
+                    },
+                    "CursorScheme.PtrEnhApply": {
+                        "schema": {
+                            "title": "Cursor locator display mode",
+                            "description": "Changes when to display the cursor locator.",
+                            "enum": [
+                                1,
+                                3,
+                                2,
+                                4
+                            ],
+                            "enumLabels": [
+                                "Always",
+                                "When pointer is stationary",
+                                "When pointer is moving",
+                                "When modifier key is pressed"
+                            ],
+                            "default": 1
+                        }
+                    },
+                    // Bitmask Setting
```

Comment:
IMO we should not onboard bitmask settings unless we have a way to handle them, like a transform from a bunch of individual settings.  In this case we also have no idea about the safe values or default.  I would suggest removing any that we don't have defaults and values for.  If there are any with a smaller range of inputs, we can enumerate them as enums/enumLabels.

Reply:
these have all been removed except for the one with a default, minimum, and maximum